src/components/backend-ai-session-launcher.ts

```diff
@@ -2750,6 +2785,10 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
     nextButton.style.visibility = this.currentIndex == this.progressLength ? 'hidden' : 'visible';
     this.shadowRoot.querySelector('#launch-button-msg').textContent = this.progressLength == this.currentIndex ? _text('session.launcher.Launch') : _text('session.launcher.ConfirmAndLaunch');
 
+    if (this.currentIndex == 2) {
+      this._showPathDescription();
```

Comment:
It's good to show this information right away when `currentIndex` is 2. However, if the user is already aware of this, showing it every time can cause irritation. Therefore, it is better to add a function not to see each other today. It would be nice to use localStorage(cookie) or implement it the way you want.
> You can refer localStorage by `bacakend-ai-login.ts` file